### PR TITLE
Update the "I've been accepted! Now what?" section

### LINKED ISSUE #343: In the application instructions, it says to leave a bio on the TFH-Bios account, not using the dashboard.
In the application instructions, under "I've been accepted! Now what?", it tells you to leave a bio on the TFH-Bios Scratch account. Since changing and adding bios has moved to the dashboard, I think we should change it to tell you to use the dashboard. I know that the account will tell you to use the dashboard, but it's still outdated information and might be a bit of an additional hassle.

### PR DESCRIPTION
**Pull request by penguinmoose**
### Resolves:

Resolves #343

### Changes:

In the application instructions, it says to leave a bio on the TFH-Bios account, not using the dashboard. This will update the text so it says to leave a bio using the dashboard instead, with a link to the dashboard (as well as a link to the TFH list and -Accio-'s and leahcimto's Scratch profiles).

### Local Tests:

I tested the changed file on an online HTML editor and it looked okay (I tried using Github Desktop, but for some reason, it did not update).

### CODE REVIEW

**Review by leahcimto — CHANGES_REQUESTED** [review]
The last sentence of "Please only contact admins if you haven't received access to the Dashboard for 1 week." sounds like you are only allowed to contact admins after a week of your acceptance for all issues including the dashboard not working or something like that. Perhaps rewording it to something like: "If you have not received access to the dashboard yet, please wait a week before contacting admins about being added."

Do you have any thoughts on this @penguinmoose and @Accio1?

**Comment by penguinmoose** [comment]
@leahcimto Sure. I agree.

**Comment by penguinmoose** [comment]
@leahcimto @Accio1 How do I change the code in the pull request?

File: apply/index.html
Comment (Accio1) [inline apply/index.html:56]: Having the wait a week part at the end seems kind of awkward. Also, saying "before contacting admins" might not be helpful because it doesn't specify anywhere that leahcimto and I are admins.

I think two changes could fix this:
1. Change the sentence about issues to:
If you have any issues with this process or questions about the dashboard, please contact a dashboard admin (-Accio- or leahcimto) on Scratch.
2. Change the last sentence to:
If you have not received access to the dashboard by one week after your acceptance date, please contact a dashboard admin.

**Review by Accio1 — CHANGES_REQUESTED** [review]

File: apply/index.html
Comment (penguinmoose) [reply apply/index.html:56]: Sure. I'll implement it when I have time.

Comment (penguinmoose) [reply apply/index.html:56]: @Accio1 Implemented. Can you take a look at it and let me know what you think of it?

**Comment by penguinmoose** [comment]
@Accio1 @leahcimto Does this look good? If so, can you approve it?

**Comment by leahcimto** [comment]
> @Accio1 @leahcimto Does this look good? If so, can you approve it?

@penguinmoose I don't see your commit...

**Comment by penguinmoose** [comment]
@leahcimto hmm let me check

**Comment by penguinmoose** [comment]
@leahcimto yeah, I might have forgot. I’ll do it tonight.

**Comment by penguinmoose** [comment]
@Accio1 @leahcimto I made the commit. Can you take a look at it and approve it if it looks good? I already tested it on an online HTML editor.

**Review by leahcimto — APPROVED** [review]

**Comment by penguinmoose** [comment]
@Accio1, can you approve this PR?

**Comment by penguinmoose** [comment]
@Accio1, can you approve this PR?

**Comment by penguinmoose** [comment]
@Accio1 ? Can you approve?

**Comment by penguinmoose** [comment]
@Accio1, can you approve?

**Comment by penguinmoose** [comment]
@BarelySmooth Someone needs to approve it. Can you if you like it?

**Comment by leahcimto** [comment]
> @BarelySmooth Someone needs to approve it. Can you if you like it?

@penguinmoose This doesn't need any more approval, it just needs to be merged by Accio1. Wait a week or so before pinging them again.

**Comment by BarelySmooth** [comment]
> @BarelySmooth Someone needs to approve it. Can you if you like it?

Me approving wouldn’t make a difference. Only Accio and leahcimto can approve PRs (with the green checkmark), since they have write access.

**Comment by penguinmoose** [comment]
Oh, I see. Both Leahcimto and Accio need to approve it?

**Comment by BarelySmooth** [comment]
> Oh, I see. Both Leahcimto and Accio need to approve it?

This repo doesn't have a minimum number of required approvals to merge a PR. Anyone with repository edit access can merge it.

(this is getting kinda offtopic, you should probably open a discussion if you'd like to know more)

**Comment by penguinmoose** [comment]
@BarelySmooth Got it. I'll just wait a week or two.

**Comment by penguinmoose** [comment]
Okay, pinging them (one day early)!

@Accio1, can you approve and merge? This PR is still not merged.

**Comment by Accio1** [comment]
> @Accio1, can you approve this PR?



> @Accio1 ? Can you approve?



> @Accio1, can you approve?



> Okay, pinging them (one day early)!
> 
> @Accio1, can you approve and merge? This PR is still not merged.

@penguinmoose In the future, please do not do this. Requesting my review once is sufficient. I have been inactive because I have been recovering from having my wisdom teeth out and spending time with family. Pinging me four times for review in the span of less than two weeks is unnecessary.

**Review by Accio1 — APPROVED** [review]

**Pull request merged**
